spring-ai-docs/src/main/antora/modules/ROOT/pages/observabilty/index.adoc

```diff
@@ -50,6 +53,9 @@ WARNING: If you enable the inclusion of the input content in the observations, t
 
 === Chat Client Advisors
 
+The `spring.ai.chat.client.advisor` observations are recorded when a call or stream around advisors is performed. 
```

Comment:
This observation is named `spring.ai.advisor` now

Reply:
of course